Doc/library/traceback.rst

```diff
@@ -62,6 +67,10 @@ The module defines the following functions:
    .. versionchanged:: 3.5
       The *etype* argument is ignored and inferred from the type of *value*.
 
+   .. versionchanged:: 3.10
+      The *etype* parameter has been renamed to *exc* and is now
```

Comment:
Could you add a simple sentence regarding this to `Porting to Python 3.10` in the 3.10 what's new?